students/guillaume/session07/html_render.py

```diff
@@ -0,0 +1,129 @@
+
+
+class Element():
+    tag = ''
+    indent = ''
+
+    def __init__(self, content=None, **kwargs):
+        self.kwargs = kwargs
+        if content is None:
+            self.content = []
+        else:
+            self.content = [content]
+
+    def append(self, content=None, **kwargs):
```

Comment:
when you are adding more content to a tag -- is it appropriate to add attributes then? 

Probably not.

If you want to change the attributes of an existing element, you can do that by simply adding to the attribute dict:

the_element.kwargs["an_attribute"] = the_value_of_the_attribute

Reply:
And this kind of makes it clear why "kwargs" is not a great name for the attribute...